__tests__/ExpensiMark-HTML-test.js

```diff
@@ -1551,3 +1551,14 @@ test('Mention', () => {
     testString = '@user@DOMAIN.com';
     expect(parser.replace(testString)).toBe('<mention-user>@user@DOMAIN.com</mention-user>');
 });
+
+test('Test code fence within inline code', () => {
+    let testString = '`(```test```)`';
+    expect(parser.replace(testString)).toBe('&#x60;(<pre>test</pre>)&#x60;');
+
+    testString = '`test`space```block```';
+    expect(parser.replace(testString)).toBe('<code>test</code>space<pre>block</pre>');
+    
+    testString = '```block```space`test`';
+    expect(parser.replace(testString)).toBe('<pre>block</pre>space<code>test</code>');
```

Comment:
Hmm, the output of these strings in slack is different from what we're expecting